README.md

```diff
@@ -45,6 +45,8 @@ try {
   // you may print it to console
   // or return it via an API
   console.log(validationError);
+  // and in case you want to get the actual error message
+  console.log(validationError.message);
```

Comment:
Of course! Just ping me if you need any help, I'd be glad to contribute

Reply:
Thank you for your patience @ogabrielp - the last few weeks have been hectic at work and I didn't have the time to investigate this in depth as I wanted to.

tl;dr your PR is correct.

I would only like to suggest that you slightly change the docs to make use of `toString`. _Why?_ Because it introduces an abstraction layer that would allow us to make further changes to the underlying method in the future.

```suggestion
  // and in case you want to get the actual error message
  console.log(validationError.toString());
```

### Long story

Apparently, `toString` is NOT automatically called with `console.log` as I originally thought.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/toString

Interestingly enough, any of the following would automatically trigger `toString`...

```typescript
`${validationError}`
```

```typescript
'' + validationError
```

That said, your comment is correct.

Reply:
No worries, I'm no stranger to hectic weeks. Thank you for taking the time to review this over the holidays, by the way. I totally agree with your suggestion: using `message` directly was bothering me too, just pushed a new commit fixing that.